e2e/cypress/integration/messaging/message_reply_scrollable_spec.js

```diff
@@ -13,6 +13,9 @@
 import * as TIMEOUTS from '../../fixtures/timeouts';
```

Comment:
I've added tests here because this test contained required fixtures. Loading them again in separate test would cost additional 70 seconds build time.